app/stores/walletModeStore.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-underscore-dangle */
+// @flow
+import db from '../services/db'
+
+export type WalletMode = "Light" | "Full";
+const WALLET_MODE_KEY = 'wallet.mode'
+
+class WalletModeStore {
+    _mode: WalletMode
+    constructor() {
+      this._mode = db.get(WALLET_MODE_KEY).value() || 'Light'
+    }
+
+    get mode() {
+      return this._mode
+    }
+
+    set mode(mode: WalletMode) {
+      this._mode = mode
+      db.set(WALLET_MODE_KEY, mode).write()
+    }
+
+    isFullNode() {
+      return this._mode === 'Full'
+    }
+
+    get modes() {
```

Comment:
I prefer to use `mode` and `otherMode`, like we do in `networkStore` for `chain` and `otherChain`.

Reasons being:
1. currently, consumer of `modes` relies on unexplicit order of the modes. I prefer to explicitly see `mode`/`otherMode`
2. Scanning the `modes` getter requires additional investigation to know what it is used for (that's what I've just done - in order to understand how it's used, I've run a search across the project).
3. To be consistent with the codebase (same mechanism/logic as chain/otherChain)

Thoughts?

Reply:
The idea was to use a tuple type - but I can see how it adds some cognitive overhead - particularly with point 3. I'll update